script/bootstrap_travis_arm

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
follow this file `https://github.com/openthread/openthread/blob/master/script/bootstrap`?